release-notes/v0.9.0.md

```diff
@@ -135,7 +135,7 @@ We’d like to encourage more developers to join us and improve Java development
 
 ## Work with JAR Files Directly
 
-If you just started to learn Java and don't have any clue what Maven or Gradle is, we got you back. Now VS Code Java lets you work with JAR files directly without any build tools. Go to `JAVA DEPENDENCIES` view, find the `Referenced Libraries` node and click the `+` icon:
```

Comment:
I suggest not to change the name. Because at that time (v0.9.0), the explorer was called "JAVA DEPENDENCIES". It's wired to change content of historical release notes.

Reply:
How about adding a note to show that we're using `JAVA PROJECTS` since a specific version?